packages/block-library/src/social-links/edit.js

```diff
@@ -104,10 +123,11 @@ export function SocialLinksEdit( props ) {
 
 	const blockProps = useBlockProps( { className } );
 	const innerBlocksProps = useInnerBlocksProps( blockProps, {
-		placeholder: isSelected ? SelectedSocialPlaceholder : SocialPlaceholder,
+		placeholder: ! isSelected && SocialPlaceholder,
 		templateLock: false,
 		orientation: attributes.layout?.orientation ?? 'horizontal',
 		__experimentalAppenderTagName: 'li',
+		renderAppender: hasAnySelected && InnerBlocks.ButtonBlockAppender,
```

Comment:
There's a focus loss when pressing escape from the appender popover.

Reply:
Thanks. I'll take a look.

Reply:
I can't seem to replicate this...how do you do it?

How do you track focus? I'm using `document.activeElement`.

Reply:
- Navigate to the appender
- Press Enter
- Focus is within the popover
- Press Escape
- Focus is...? It should be visible and on the appender again since that's what opened the popover.

Reply:
Oh man. I missed the word "appender". My bad. I was looking at the URL popover 🤦

Reply:
I pushed a commit which aims to restore focus to the appender once the quick inserter is closed. LMK what you think 🙏

Reply:
Fixed! Thank you!